requirements.txt

```diff
@@ -1,3 +1,8 @@
 numpy>=1.18.3
 scipy>=1.7
-scikit-learn>=1.2.0
\ No newline at end of file
+scikit-learn>=1.2.0
+
+-e .
```

Comment:
This was new to me

Reply:
This identifies basic dependencies and developer dependencies.  Anything below `-e .` will only get checked if `-r requirements.txt` is explicitly invoked.